frameworks/kafka/universe/config.json

```diff
@@ -1258,7 +1353,7 @@
           "title": "ssl.endpoint.identification.enabled",
           "description": "By default enables the hostname verification when TLS connections are established between Kafka brokers. To disable hostname verification leave this option unchecked",
           "type": "boolean",
-          "default": true
+          "default": false
```

Comment:
What's the reason for disabling this here?

Reply:
Enabled. Disabled this for testing purposes as the testing certs were self signed.